spyder/plugins/completion/languageserver/confpage.py

```diff
@@ -713,16 +713,26 @@ def setup_page(self):
                                      "definition"),
                                    'jedi_definition/follow_imports')
         show_signature_box = newcb(_("Show calltips"), 'jedi_signature_help')
+        automatic_completion_box = newcb(_("Show completions on the fly"),
+                                         'automatic_completions',
+                                         section='editor')
+        completion_hint_box = newcb(_("Show completion details in hint"),
```

Comment:
Let's change the text of this checkbox to simply be

    Show completion details